shared/getResinOS/_default.md

```diff
@@ -2,13 +2,15 @@
 <!--add an anchor here to redirect old links-->
 <a name="adding-your-first-device"></a>
 
-To connect with resin.io, your **{{ $device.name }}** will need a resinOS image that is configured for your application and your network. Start by clicking *Add device* in your application dashboard:
+To connect with resin.io, your **{{ $device.name }}** will need a resinOS image that is configured for your device type, application, and network. Start by clicking *Add device* in your application dashboard:
 
-<img src="/img/common/app/add_device.png" width="100%">
+<img src="/img/{{ $device.id }}/add-device.png" width="100%">
 
-A list of available resinOS versions is shown in the dropdown. In general, the most recent version is recommend. You can also select whether you would prefer a *Development* or *Production* edition with a toggle:
+For most applications, you will have the option to select a device type. By default, the device type you chose when you first created the application will be selected. Applications can, however, support any devices that share the same architecture, so you can choose another device type if needed.
 
-<img src="/img/common/app/version.png" width="60%">
+After selecting a device type, you will see a list of available resinOS versions. In general, the most recent version is recommend. You can also select whether you would prefer a *Development* or *Production* edition with the respective toggle:
+
+<img src="/img/{{ $device.id }}/device-type.png" width="60%">
```

Comment:
NIT: Maybe `device-type.png` isn't that representative of what the image shows. It sounds like a simple logo to me.

Reply:
I think you're suggesting that maybe `add-new-device.png` would be a better name for these images than `device-type.png` ?

Reply:
I'll change to `os-options.png`, I think that fits best for what the image shows